TROUBLESHOOTING.md

```diff
@@ -3,26 +3,74 @@ Troubleshooting
 
 Indoor maps can be complex! On this page you'll find some common problems and fixes for them.
 
-1. [Failed Schema Validation](#failed-schema-validation)
-1. [Failure to find indoor routes](#failure-to-find-indoor-routes)
+- Indoor Map Submissions
+	- [Failed Schema Validation](#failed-schema-validation)
+	- [Multipolygon Errors](#multipolygon-errors)
+- Indoor Navigation Paths
+	- [Failure to find indoor routes](#failure-to-find-indoor-routes)
+- Indoor Assets
+	- [Importing: Failed to find any matching inserts](#importing-failed-to-find-any-matching-inserts)
+
+
+## Indoor Map Submissions
 
 ### Failed Schema Validation 
-The submission service will email you if your map doesn't meet the specification for our geoJSON.
+
+The submission service will email you if your map doesn't meet the [specification for our GeoJSON](FORMAT.md).
 
 Common problems are:
+
 1. Incorrect CRS. The CRS (Coordinate Reference System) must be WGS84 ([EPSG: 3857](http://spatialreference.org/ref/sr-org/6864/)).  This can be set at the time of export from QGIS.
-1. Missing ids.  The ids for features can be generated by following the instructions in [the tutorial](TUTORIAL.md#user-content-generate-feature-ids).
+2. Missing ids.  The ids for features can be generated by following the instructions in [the tutorial](TUTORIAL.md#user-content-generate-feature-ids).
+
+
+### Multipolygon errors
```

Comment:
I didn't realise this section existed here!  Please factor out the content from the bottom of TUTORIAL.md and TUTORIAL-TOOL.md (which is updated by my PR yesterday, so should replace the content here) and replace with a link to this section.